src/component/application/ApplicationList.tsx

```diff
@@ -1,36 +1,20 @@
 import React, { useEffect, useMemo, useState } from 'react';
-import PropTypes from 'prop-types';
 import { CircularProgress } from '@material-ui/core';
 import { Warning } from '@material-ui/icons';
 
 import { AppsLinkList, styles as commonStyles } from '../common';
 import SearchField from '../common/SearchField/SearchField';
 import PageContent from '../common/PageContent/PageContent';
 import HeaderTitle from '../common/HeaderTitle';
+import useApplications from '../../hooks/api/getters/useApplications/useApplications';
 
-const Empty = () => (
-    <React.Fragment>
-        <section style={{ textAlign: 'center' }}>
-            <Warning /> <br />
-            <br />
-            Oh snap, it does not seem like you have connected any applications.
-            To connect your application to Unleash you will require a Client
-            SDK.
-            <br />
-            <br />
-            You can read more about how to use Unleash in your application in
-            the{' '}
-            <a href="https://docs.getunleash.io/docs/sdks/">documentation.</a>
-        </section>
-    </React.Fragment>
-);
-
-const ClientStrategies = ({ fetchAll, applications }) => {
+const ApplicationList = () => {
+    const { applications, refetchApplications } = useApplications();
     const [filter, setFilter] = useState('');
-
     useEffect(() => {
-        fetchAll();
-    }, [fetchAll]);
+        refetchApplications();
```

Comment:
I don't think this is necessary. The useSWR hook should automatically fetch the applications once it is initialised.